documentation/construction.md

```diff
@@ -0,0 +1,66 @@
+---
+Make:
+  base plate:
+    template: printing.md
+    stl-file: ../build/printed_components/baseplate.stl
+    stlname: baseplate.stl
+    material: PLA
+    weight-qty: 50g
+  top plate:
+    template: printing.md
+    stl-file: ../build/printed_components/topplate.stl
+    stlname: topplate.stl
+    material: PLA
+    weight-qty: 50g
+  rack legs:
+    template: printing.md
+    stl-file: ../build/printed_components/beam.stl
+    stlname: beam.stl
+    material: PLA
+    weight-qty: 60g
+  stuff shelf:
+    template: printing.md
+    stl-file: models/Stuff_Shelf_1.0.stl
+    stlname: Stuff_Shelf_1.0.stl
+    material: PLA
+    weight-qty: 50g
+---
+
+# Construct the and populate the rack
```

Comment:
I'm not sure if this is supposed to say "Construct the case" or "Construct the shelves" or something else.

Reply:
I am pretty sure it was just meant to be "Construct and populate the rack"

Reply:
Fixed as above